tools/DARIAH-DE-Geo-Browser-KML.json

```diff
@@ -0,0 +1,37 @@
+{
+    "task": "Visualisation",
+    "deployment": "development",
+    "name": "DARIAH-DE Geo-Browser (KML)",
+    "logo": "dariah.svg",
```

Comment:
Same naming error as in the other json file

Reply:
Thanks again :-) Also fixed!